docs/tutorials/excel-tutorial-create-custom-functions.md

```diff
@@ -143,6 +143,8 @@ Integrating data from the Web is a great way to extend Excel through custom func
         // Note: in case of an error, the returned rejected Promise
         //    will be bubbled up to Excel to indicate an error.
     }
+    
+> [!NOTE] In the January Insiders 1901 Build, there is a bug preventing fetch calls from executing which will result in seeing #VALUE!. > To workaround, this please use the [XMLHTTPRequest API](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-runtime#requesting-external-data) to make the web request.
```

Comment:
```suggestion
> [!NOTE] In the January Insiders 1901 build, a known bug prevents `fetch` calls from executing. As a result, cells may instead show #VALUE!. As a workaround, please use the [XMLHTTPRequest API](https://docs.microsoft.com/office/dev/add-ins/excel/custom-functions-runtime#requesting-external-data) to make the web request instead.
```

Reply:
Maybe we should put this in the changelog section instead of here? Or in both places?